nengo/tests/test_dists.py

```diff
@@ -68,19 +68,36 @@ def test_exponential(scale, shift, high, rng):
         assert abs(np.mean(samples - shift) - scale) < ci
 
 
-@pytest.mark.parametrize("dimensions", [0, 1, 2, 5])
-def test_hypersphere(dimensions, rng):
+@pytest.mark.parametrize("min_magnitude,dimensions",
+                         [(0, 1), (0, 2), (0, 5),
+                          (0.5, 1), (0.5, 2), (0.5, 5), (2, 1), (-2, 1)])
+def test_hypersphere(min_magnitude, dimensions, rng):
```

Comment:
This is a weird test since it's checking a bunch of different things. Also, we lost the test case where `dimensions == 0`. I'd almost rather keep the original test as-is, and add a second test for the `min_magnitude` parameter.